### PR TITLE
Migrate to zap

### PR DESCRIPTION
**Pull request by dim**

### CODE REVIEW

**Comment by dim** [comment]
@mxmCherry @115100 @jpmedforth

File: log/kafka/kafka.go [hunk 1]
```diff
@@ -1,179 +0,0 @@
-/*
```
Comment (mxmCherry) [inline log/kafka/kafka.go:1]: That's totally gone? No kafka logging support out of the box anymore?

File: log/log.go [hunk 2]
```diff
@@ -1,71 +1,118 @@
...
+func Warn(args ...interface{})  { std.Warn(fmt.Sprint(args...)) }
+func Fatal(args ...interface{}) { std.Fatal(fmt.Sprint(args...)) }
+
+func Debugf(format string, args ...interface{}) { std.Debug(fmt.Sprintf(format, args...)) }
+func Errorf(format string, args ...interface{}) { std.Error(fmt.Sprintf(format, args...)) }
+func Infof(format string, args ...interface{})  { std.Info(fmt.Sprintf(format, args...)) }
+func Warnf(format string, args ...interface{})  { std.Warn(fmt.Sprintf(format, args...)) }
+func Fatalf(format string, args ...interface{}) { std.Fatal(fmt.Sprintf(format, args...)) }
+
+func Debugln(args ...interface{}) { std.Debug(fmt.Sprintln(args...)) }
```
Comment (mxmCherry) [inline log/log.go]: Should `*ln` variant be really available here? IMO, better don't make consumer think about newlines.

File: log/log.go [hunk 3]
```diff
@@ -1,71 +1,118 @@
...
+func Errorln(args ...interface{}) { std.Error(fmt.Sprintln(args...)) }
+func Infoln(args ...interface{})  { std.Info(fmt.Sprintln(args...)) }
+func Fatalln(args ...interface{}) { std.Fatal(fmt.Sprintln(args...)) }
+func Warnln(args ...interface{})  { std.Warn(fmt.Sprintln(args...)) }
+
+func Debugw(msg string, fields ...zapcore.Field) { std.Debug(msg, fields...) }
```
Comment (mxmCherry) [inline log/log.go]: What `*w` stands for? "With"? :)

**Review by mxmCherry — COMMENTED** [review]
Some usage-related questions

File: log/kafka/kafka.go (hunk 1 above)
Comment (dim) [reply log/kafka/kafka.go:1]: as long as we don't need it ... we can always bring it back

File: log/log.go (hunk 2 above)
Comment (dim) [reply log/log.go]: here's the difference: https://play.golang.org/p/2nWgpe-aOn

File: log/log.go (hunk 3 above)
Comment (dim) [reply log/log.go]: just following zap conventions, but yes, probably: https://godoc.org/go.uber.org/zap#SugaredLogger.Infow

File: log/log.go (hunk 2 above)
Comment (mxmCherry) [reply log/log.go]: Doesn't look like killer feature, tbh, but, probably, makes sense for others

**Pull request merged**